supertokens_python/recipe/emailpassword/emaildelivery/service/smtp/service_implementation/email_verification_implementation.py

```diff
@@ -0,0 +1,33 @@
+# Copyright (c) 2021, VRAI Labs and/or its affiliates. All rights reserved.
+#
+# This software is licensed under the Apache License, Version 2.0 (the
+# "License") as published by the Apache Software Foundation.
+#
+# You may not use this file except in compliance with the License. You may
+# obtain a copy of the License at http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+# WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+# License for the specific language governing permissions and limitations
+# under the License.
+
+from typing import Any, Dict
+
+from supertokens_python.ingredients.emaildelivery.service.smtp import (
+    GetContentResult, ServiceInterface, SMTPServiceConfigFrom)
+from supertokens_python.recipe.emailpassword.types import (
+    TypeEmailPasswordEmailDeliveryInput,
+    TypeEmailVerificationEmailDeliveryInput)
+
+
+class ServiceImplementation(ServiceInterface[TypeEmailVerificationEmailDeliveryInput]):
+    def __init__(self, email_verification_service_implementation: ServiceInterface[TypeEmailPasswordEmailDeliveryInput]) -> None:
```

Comment:
This variable should be called `email_password_service_implementation`? (See the type of it)